front/src/routes/integration/all/withings/device-page/Device.jsx

```diff
@@ -0,0 +1,159 @@
+import { Text, Localizer } from 'preact-i18n';
+import { Component } from 'preact';
+import cx from 'classnames';
+import get from 'get-value';
+import dayjs from 'dayjs';
+import relativeTime from 'dayjs/plugin/relativeTime';
+
+dayjs.extend(relativeTime);
+
+import { DEVICE_FEATURE_TYPES } from '../../../../../../../server/utils/constants';
+import { RequestStatus, DeviceFeatureCategoriesIcon } from '../../../../../utils/consts';
+
+class WithingsDeviceBox extends Component {
+  refreshDeviceProperty = () => {
+    if (!this.props.device.features) {
+      return null;
+    }
+    const batteryLevelDeviceFeature = this.props.device.features.find(
+      deviceFeature => deviceFeature.type === DEVICE_FEATURE_TYPES.SENSOR.INTEGER
```

Comment:
You should filter by category instead of type (for example, see the Bluetooth, Xiaomi or Zwave service).

Reply:
I fix it